examples/bpk-component-graphic-promo/examples.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import BpkGraphicPromo, {
+  TEXT_ALIGN,
+} from '../../packages/bpk-component-graphic-promo';
+
+const image =
+  'https://content.skyscnr.com/m/4933408c167ee099/original/MERCH-131-Graphic-Promo-2096x800.png';
+const sponsor = {
+  label: 'Sponsored',
+  logo: 'https://www.easyjet.com/ejcms/cache/medialibrary/Images/Global/Main-III/Logo/ej-transparent-x.png',
+  altText: 'easyJet',
+};
+const ctaText = 'Learn more';
+const onClick = () => {
+  window.location.href = 'https://www.skyscanner.net';
+};
+
+const kicker = 'Kicker';
+const headline = 'Ride your wave';
+const strapline =
+  'Portugal and 6 more countries have just been added to the UK travel green list';
+
+const DefaultExample = () => (
+  <BpkGraphicPromo
+    kicker={kicker}
+    headline={headline}
+    strapline={strapline}
+    image={image}
+    sponsor={sponsor}
+    ctaText={ctaText}
+    onClick={onClick}
+    textAlign={TEXT_ALIGN.start}
+    invertVertically={false}
```

Comment:
Does this need to be explicitly stated? Would the absence of this prop have the same effect?

Reply:
Yes, it would be equivalent. I just made the `prop` required in the `PropTypes`, because the designs specified two variants. I will make it optional and just not pass it, if `false`. 🙂